src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -74,6 +77,10 @@ const SelfExclusion = (() => {
                 }
                 return;
             }
+            BinarySocket.send({ get_limits: 1 }).then((data) => {
+                max_limits.account_balance = data.get_limits.account_balance ? data.get_limits.account_balance : null;
+                max_limits.open_positions = data.get_limits.open_positions ? data.get_limits.open_positions : null;
```

Comment:
```suggestion
                max_limits = {
                    ...(data.get_limits.account_balance &&  { account_balance: data.get_limits.account_balance }),
                    ...(data.get_limits.open_positions &&  { open_positions: data.get_limits. open_positions }),
                };
```

Reply:
just a nitpick for easier additions in the future, if you think the current one is better then we can stick to that and move this forward.